app/controllers/files_controller.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+class FilesController < ApplicationController
+  include Authenticatable
+
+  def index
+    # TODO: Implement the main logic.
+    Api::Files.all({ access_token: current_access_token })
```

Comment:
実験してはいないのだけど、たぶん Hash オブジェクトじゃなく、キーワード引数で渡しても動くんじゃないかなあ
```suggestion
    Api::Files.all(access_token: current_access_token)
```

Reply:
むむ、確かに。

Reply:
キーワード引数でも動きました。修正します。